clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/tests/WgsUbamTests.scala

```diff
@@ -230,6 +232,13 @@ trait WgsUbamTests { self: BaseIntegrationSpec =>
       )
       sampleResults <- Unmarshal(sampleResponse)
         .to[Seq[TransferWgsUbamV1QueryOutput]]
+      rpIdResponse <- runClient(
+        ClioCommand.queryWgsUbamName,
+        "--research-project-id",
```

Comment:
Is there a way to avoid using a magic string? We can import caseapp and the class that implements this option, can that be used here?

Reply:
I don't think there's a way to avoid it right now (if we can, it'll be from writing custom shapeless code to say "give me the string for the field in this class with the symbol researchProjectId"). case-app doesn't provide any sort of reverse-parsing capabilities to turn objects into equivalent cmd-line args.